src/__tests__/starWarsSchema.js

```diff
@@ -170,6 +170,29 @@ const humanType = new GraphQLObjectType({
         'The friends of the human, or an empty list if they have none.',
       resolve: (human) => getFriends(human),
     },
+    friendsAsync: {
+      type: GraphQLList(characterInterface),
+      description:
+        'The friends of the droid, or an empty list if they have none. Returns an AsyncIterable',
+      args: {
+        errorIndex: { type: GraphQLInt },
+      },
+      async *resolve(droid, { errorIndex }) {
+        const friends = getFriends(droid);
+        let i = 0;
+        for (const friend of friends) {
+          // eslint-disable-next-line no-await-in-loop
+          await new Promise((r) => setTimeout(r, 1));
```

Comment:
Why do you need `setTimeout`?
A promise is always processed on a next tick even if it already resolved.

Reply:
Leftover from when I was trying to time results for tests combining `@stream` and `@defer`. Not needed any more and removed from this PR